packs/chef/actions/chef_install.py

```diff
@@ -33,15 +33,15 @@ def install(self, install_method, options):
     def execute(self):
         self.parser = shell.CmdlineParser(self.cmdline_options)
         options = self.parser.parse()
-        install_method = [i for i in options.keys() if i in self.SUPPORTED_METHODS ]
+        install_method = options['method']
 
-        if len(install_method) > 1:
-            sys.stderr.write("You can't use more than one installation method!\n")
-            sys.stderr.write("Choose one of the following methods: %s\n" % ', '.join(self.SUPPORTED_METHODS))
+        if not (install_method in self.SUPPORTED_METHODS):
```

Comment:
A bit more pythnoic would be to say `if install_method not in self.SUPPORTED_METHODS`